app/views/announcements/_announcement.html.erb

```diff
@@ -1,9 +1,12 @@
 <div id="<%= dom_id announcement %>" class="announcement-container bg-info-subtle rounded shadow-lg p-4 mb-3">
   <div class="d-flex flex-row align-items-start mb-2">
     <!-- User Avatar -->
-    <div class="me-3">
-      <%= image_tag User.find_by(uid: announcement.googleUserID).avatar_url, alt: "User Avatar", class: "rounded-circle", style: 'width: 60px; height: 60px; object-fit: cover;' %>
-    </div>
+    <% announcement_user = User.find_by(uid: announcement.googleUserID) %>
+    <% if announcement_user.present? %>
```

Comment:
What happens here if the announcement_user is not present, maybe we should have a placeholder image. Maybe like https://static.thenounproject.com/png/574748-200.png

This is up to you, however, i think we should show some avatar image for the sake of consistency even if the user is deleted

Reply:
Yea good catch. I remember thinking about that, but decided to just remove the div if there's no one logged in, but I will switch to a default avatar